preprocessing/biotoolbox/contact_map_builder.py

```
@@ -1,321 +1,321 @@
+        self.chains[chain_name]['method'] = method
+
+    def with_final_seq_for_chain(self, chain_name, final_seq):
+        self.chains[chain_name]['final-seq'] = final_seq
+
+
+def correct_residue(x, target):
+    try:
+        sl = seq1(x.resname, undef_code='-', custom_map=protein_letters_3to1 )
+        if sl == target:
+            return True
+        return False
```

Ruff (0.8.2)

Return the condition `sl == target` directly

Replace with `return sl == target`

(SIM103)